module-api/src/main/java/ru/feathercore/moduleapi/ModuleLoader.java

```diff
@@ -62,4 +62,15 @@
      * @apiNote modifications to returned collection should not have side-effect on this module loader
      */
     Collection<? extends M> getModules();
+
+    /**
+     * Checks whether module of given class was loaded by this module loader.
+     *
+     * @param module class of module to be checked.
+     * @return true/false.
+     */
+    default boolean isModuleLoaded(Class<? extends M> module) {
+        return getModules().stream().anyMatch(mod -> module.isAssignableFrom(mod.getClass()));
```

Comment:
Provide default implemention in `SimpleModuleLoader`

Reply:
Why?

Reply:
getModules() is contracted to return unmodifiable view of loaded modules and so some implementations may provide a more efficient implementation. A default one may be kept, but an overriden equivalent should definitely be added to default implementation